lib/matplotlib/axes/_base.py

```diff
@@ -3340,7 +3341,10 @@ def _set_view_from_bbox(self, bbox, original_view, direction='in',
 
         lastx, lasty, x, y = bbox
 
-        x0, y0, x1, y1 = original_view
+        if original_view is None:
+            original_view = self._get_view()
+        else:
+            x0, x1, y0, y1 = original_view
```

Comment:
Thanks, I have just spent the last few hours pouring over the colorbar inverse code so I have my mind in code-reading mode.

Though as I say in my comment below I don't see that this part of the code does actually does anything, it looks dead...